tests/standalone_tests/test_local_global_step_logic.py

```diff
@@ -36,8 +36,19 @@ def get_parametrization_list_success_cases():
     "algorithm_name, request_dict",
     get_parametrization_list_success_cases(),
 )
-def test_local_global_step_algorithms(algorithm_name, request_dict):
-    algorithm_url = algorithms_url + "/" + algorithm_name
+def test_local_global_step_algorithms(
```

Comment:
Only the `edsd0` dataset is used so I don't think more than one localnode is needed.